docker-compose-config.py

```diff
@@ -2,8 +2,11 @@
 import string
 from typing import Optional
 
-import yaml
-from yaml.loader import FullLoader
+try:
+    import yaml
```

Comment:
I actually keep getting an error here. Apparently a common issue:

https://github.com/ipython-contrib/jupyter_contrib_nbextensions/issues/590
https://github.com/yaml/pyyaml/issues/291

Reply:
Hm, made `pyyaml` installation use `python3 -m pip` instead of `pip3`, so maybe that fixes this inconsistency here?